ansible/roles/atmo-common/tasks/main.yml

```diff
@@ -36,7 +37,8 @@
     - timezone
 
 - name: update timezone manually
-  shell: /bin/su {{ item }} -c export TZ={{ TIMEZONE }}
+  become_user: '{{ item }}'
```

Comment:
fwiw, I don't believe this shell command has any effect. `export` only affects the current and child processes.

Reply:
Should I just delete this task?